### PR TITLE
ClusterLoader - 5x100 density test update

### PR DESCRIPTION
**Pull request by krzysied**
Temporally changing test to make 5x  create latency pods iterations. Required to compare different latency pods creation approaches.

### CODE REVIEW

File: clusterloader2/testing/density/config.yaml [hunk 1]
```diff
@@ -61,7 +61,8 @@ steps:
     Params:
       action: gather
 - name: Creating saturation pods
-# Create latency pods
+# TODO: Change this explicite iterations when pod startup problem is fixed!
```
Comment (wojtek-t) [inline clusterloader2/testing/density/config.yaml]: Ohh man - how ugly that is...

Can you at least open an issue to trigger reverting that?

Comment (wojtek-t) [reply clusterloader2/testing/density/config.yaml]: I meant "to not forget about reverting" :-)

**Comment by wojtek-t** [comment]
BTW - I really doubt this is related. But you can try...

File: clusterloader2/testing/density/config.yaml (hunk 1 above)
Comment (krzysied) [reply clusterloader2/testing/density/config.yaml]: Issue #310

**Comment by krzysied** [comment]
@wojtek-t After running some local test I observed ~0.5s difference. I'm curious if there will be difference while using test environment.

**Comment by wojtek-t** [comment]
Well - there are differences between runs (especially if you're running from your desktop). So I still doubt it's that. But let's see.

/lgtm

**Pull request merged**